backend/layers/processing/requirements.txt

```diff
@@ -8,5 +9,6 @@ psycopg2-binary>=2.8.5
 pyarrow>=1.0
 pydantic>=1.9.0
 python-json-logger
+scanpy==1.6.0
```

Comment:
nit: for some reason this is pinned to 1.6.0 already, but its pinned in the processing dockerfile directly as a pip3 install scanpy==1.6.0 step. We should just keep one (probably the one here)